packages/input/src/ux-input.ts

```diff
@@ -8,12 +8,15 @@ import { UxInputTheme } from './ux-input-theme';
 import '@aurelia-ux/core/components/ux-input-component.css';
 // tslint:disable-next-line: no-submodule-imports
 import '@aurelia-ux/core/components/ux-input-component--outline.css';
+import { UxDefaultInputConfiguration } from './ux-default-input-configuration';
 
 export interface UxInputElement extends HTMLElement {
   value: any;
 }
 
-@inject(Element, StyleEngine)
+export type InputVariant = 'filled' | 'outline';
```

Comment:
maybe this type should be exported from core, as we have `InputComponent` from there

Reply:
done